pytry2.py

```diff
@@ -7,136 +7,177 @@
 import jaraco.logging
 import time
 
-KEY_MAPPINGS = {
+KEY_MAPPINGS_TUPLE = [
```

Comment:
I essentially changed this to be a list of tuples, and then when main is called I convert it into a dict and ensure that the keys match